integration-tests/vertx-web/pom.xml

```diff
@@ -0,0 +1,110 @@
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>native-image</id>
+            <activation>
+                <property>
+                    <name>native</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <skipTests>${native.surefire.skip}</skipTests>
+                        </configuration>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                                <configuration>
+                                    <systemPropertyVariables>
+                                        <native.image.path>${project.build.directory}/${project.build.finalName}-runner</native.image.path>
+                                    </systemPropertyVariables>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>io.quarkus</groupId>
+                        <artifactId>quarkus-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>native-image</id>
+                                <goals>
+                                    <goal>native-image</goal>
+                                </goals>
+                                <configuration>
+                                    <cleanupServer>true</cleanupServer>
+                                    <enableHttpUrlHandler>true</enableHttpUrlHandler>
+                                    <!-- Include the server certificate -->
+                                    <additionalBuildArgs>
+                                        <additionalBuildArg>-H:EnableURLProtocols=http,https</additionalBuildArg>
+                                    </additionalBuildArgs>
+                                    <graalvmHome>${graalvmHome}</graalvmHome>
+                                    <enableAllSecurityServices>true</enableAllSecurityServices>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
```

Comment:
Can we please not use this old style for new integration tests? It's probably going to go away soon and best minimize the work when we do :)

Setting:

```
            <properties>
                <quarkus.package.type>native</quarkus.package.type>
            </properties>
```

is all that needs to be done in the `native` Maven profile in order to produce a native binary

Reply:
@geoand Shouldn't we update all integration tests in a separate PR then? I copied this from `vertx-http`.

Reply:
Yes, at some point we will have to. My point here is that since we are adding a new IT, let's not add extra work for our future selves :)

Reply:
`quarkus-integration-test-spring-web` also uses the old style ;-)

Reply:
Yup, it's kinda old as well and needs to be fixed :P

Reply:
Fixed.